src/main/java/com/google/research/bleth/simulator/Board.java

```diff
@@ -1,15 +1,16 @@
 package com.google.research.bleth.simulator;
 
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkNotNull;
-
 import com.google.common.collect.ArrayTable;
 import com.google.common.collect.ImmutableList;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+import static com.google.common.base.Preconditions.checkArgument;
```

Comment:
Static imports should be before dynamic imports.
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing